src/components/Card/LessonCard.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react'
+import PlayButton from '../Button/PlayButton'
+import Card from './'
+
+export const LessonMeta = ({meta}) => {
+  return (
+    <div className='flex items-center gray'>
+      <img src={meta.langImg} className='w2' alt='' />
+      <i className='fa fa-clock-o o-60 f5 ml3' />
+      <div className='w3 ml2 o-60 f6'>{meta.videoLength}</div>
+    </div>
+  )
+}
+LessonMeta.propTypes = {
+  meta: PropTypes.object.isRequired
```

Comment:
@trevordmiller For the time being `meta` is just the shape of the `testData`, not the shape of the real API response